src/main.ts

```diff
@@ -72,7 +72,7 @@ async function start () {
             success: '#1a9031', // same as $app-green
             error: '#d3272c', // same as $app-red
             gray7: '#495057', // same as $gray7
-            gray9: '#495057' // same as $gray9
+            gray9: '#212529' // same as $gray9
```

Comment:
Not sure why this was wrong -- and it feels risky to fix it since many pages use gray9 -- but now this is correct and any pages that are now wrong can be updated in a future ticket if needed.